routes/socket/user-events.js

```diff
@@ -2194,6 +2194,31 @@ module.exports.handleModerationAction = (socket, passport, data, skipCheck, modU
 						if (err) socket.emit('sendAlert', `IP clear failed:\n${err}`);
 					});
 					break;
+				case 'modPeekVotes':
+					const gameToPeek = games[data.uid];
+					let output = '';
+					if (gameToPeek && gameToPeek.private && gameToPeek.private.seatedPlayers) {
+						const playersToCheckVotes = gameToPeek.private.seatedPlayers;
+						playersToCheckVotes.map(player => {

```

Comment:
`[].map` should really be used for assignment, you want `[].forEach` here.  NBD.